### PR TITLE
Add Favourites Page

### LINKED ISSUE #10: Add Favorite Page
We want to change the download section in the design into a favourites section.

image

To do:
 
- [x] Add Favourites Page
- [x] implement series-card component
- [x] implement component nav-bar
- [x] change download icon in NavBar
- [x] change 'watch now' button to 'watch later/favourite' button (in MovieDetails and TVShowDetails page)
- [x] change download icon in episode card component

### PR DESCRIPTION
**Pull request by infoucarsinan**
Fixes #10
-Add Favourites Page
-implement series-card component
-implement component nav-bar

### CODE REVIEW

**Review by lajanzen — APPROVED** [review]
LGTM 🚀

**Review by J-Bannach — APPROVED** [review]
lgtm 💯

File: src/app/pages/FavouritePage/FavouritePage.module.css [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.container {
+  padding-bottom: 6em;
+}
+.main {
+  display: grid;
+  gap: 2em;
+  justify-items: center;
+  padding-left: 4em;
+  padding-right: 4em;
```
Comment (SGierszewski) [inline src/app/pages/FavouritePage/FavouritePage.module.css:9]:
```suggestion
  padding-left: 4em;
  padding-right: 4em;
```
```suggestion
  padding-left: 4em;
  padding-right: 4em;
```

Comment (SGierszewski) [inline src/app/pages/FavouritePage/FavouritePage.module.css]: Can't this be shortened to `padding: 0 4em;`?

**Review by SGierszewski — COMMENTED** [review]
Good job, thank you! 🚀

**Review by Dev-dfm — APPROVED** [review]
lgtm 💯

**Review by dhnazarenko — APPROVED** [review]
lgtm 🚀

**Review by Niklas-Schaefers — APPROVED** [review]
lgtm 🚀

**Review by tmh9x — APPROVED** [review]

**Review by daebert — APPROVED** [review]
lgtm

**Review by roabram — APPROVED** [review]
lgtm

**Review by lmachens — APPROVED** [review]

**Pull request merged**